OurUmbraco.Site/Views/MacroPartials/Projects/ProjectDetails.cshtml

```diff
@@ -271,6 +274,11 @@
                                             </a>
                                         </h3>
                                         <small>@owner.Name.Split(' ').First() has @owner.Karma() karma points</small>
+                                        @if (currentMember != null && (currentMember.Id == owner.Id || contributors.ForEach(c => c.MemberId == currentMember.Id).SingleOrDefault()))
```

Comment:
Should work, but contributors.Any(c => c.MemberId == currentMember.Id) is easier, will fix it up!